scripts/covid19_india/cases_count_districts_data/test_data/expected.csv

```diff
@@ -0,0 +1,17 @@
+Date,District,districtCode,Count_MedicalTest_ConditionCOVID_19_Positive,Count_MedicalConditionIncident_COVID_19_PatientRecovered,Count_MedicalConditionIncident_COVID_19_PatientDeceased
+26-04-2020,Anantapur,502.0,53,14,4
```

Comment:
District code should be a 3 digit string

Reply:
Converted it to string. But couldn't retain codes like '079'. It gets changes to '79' but that is how it is in Thej's LGD PR. So I retained it.